### PR TITLE
VideoPress: Introduce support to enable and disable items at `VideoList` and `VideoRow`

### LINKED ISSUE #25744: VideoPress components: LocalVideosList
Image

### LINKED ISSUE #25745: VideoPress components: LocalVideoRow
Image

Image

### PR DESCRIPTION
**Pull request by renatoagds**
This one implements the handlers to `hide` or `show` columns and actions inside `VideoList` and `VideoRow`.

Closes #25744 
Closes #25745 

#### Changes proposed in this Pull Request:

* Open new props at `VideoRow` to hide the edit button and quick actions, and remove items from the screen if they don't exist.
* Open new props at `VideoList` to disable columns and already actions supported by `VideoRow`.
* Update logic to select all items at `VideoList`.
* Update `Typescript` types.

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Testing instructions:

* Start `Storybook`
* Navigate to `VideoPress/Components/Video Row`
* Interact with new `hideEditButton` and `hideQuickActions` props.
* Navigate to `VideoPress/Components/Video List`
* Interact with new `hidePrivacy`, `hideDuration`, `hidePlays` props.
* Change viewport in all cases to interact with small version too.

#### Demo

https://user-images.githubusercontent.com/1663717/187753698-3c8ce8b7-be84-4547-b822-fcc780b451ae.mov

### CODE REVIEW

File: projects/packages/videopress/src/client/admin/components/video-row/index.tsx [hunk 1]
```diff
@@ -184,6 +186,12 @@ const VideoRow = ( {
 	const showTitleLabel = ! isSmall && isEllipsisActive;
 	const showStats = ( ! showActions && ! isSmall ) || ( isSmall && expanded );
 	const showBottom = ! isSmall || ( isSmall && expanded );
+	const canExpand =
+		isSmall &&
+		( ! hideEditButton ||
+			Boolean( duration ) ||
+			Boolean( plays ) ||
```
Comment (dhasilva) [inline projects/packages/videopress/src/client/admin/components/video-row/index.tsx]: `plays` can be 0, right? In this case, `Boolean( plays )` returns `false`. If you are only checking that there is a value and 0 is a valid value (which I'm not sure here), `plays != null` would return `true` for `0`.

File: projects/packages/videopress/src/client/admin/components/video-row/stats.tsx [hunk 2]
```diff
@@ -17,10 +17,12 @@ const Stats = ( {
...
-				{ privacy }
-			</Text>
-			<Text component="div">{ duration }</Text>
+			{ Boolean( privacy ) && (
+				<Text aria-disabled={ isSmall ? 'false' : 'true' } component="div">
+					{ privacy }
+				</Text>
+			) }
+			{ Boolean( duration ) && <Text component="div">{ duration }</Text> }
 			{ Boolean( plays ) && <Text component="div">{ plays }</Text> }
```
Comment (dhasilva) [inline projects/packages/videopress/src/client/admin/components/video-row/stats.tsx]: If a video has 0 plays, the columns are shifted. `plays != null` would fix this case, but if for some reason one of the values is `null`, the column shift still happens. Maybe an empty `Text` to keep the column?
Screenshot_2022-08-31_16-34-57

Comment (renatoagds) [reply projects/packages/videopress/src/client/admin/components/video-row/stats.tsx]: Done: https://github.com/Automattic/jetpack/pull/25968/commits/3a8f2a3df8692cff5fccb67ca3954150c6b773b6

File: projects/packages/videopress/src/client/admin/components/video-row/index.tsx (hunk 1 above)
Comment (renatoagds) [reply projects/packages/videopress/src/client/admin/components/video-row/index.tsx]: Done: https://github.com/Automattic/jetpack/pull/25968/commits/3a8f2a3df8692cff5fccb67ca3954150c6b773b6

**Review by dhasilva — APPROVED** [review]
Nice work :rocket:

**Pull request merged**